exonum/src/runtime/mod.rs

```diff
@@ -414,16 +420,16 @@ impl Caller {
 pub struct ExecutionContext<'a> {
     /// The current state of the blockchain. It includes the new, not-yet-committed, changes to
     /// the database made by the previous transactions already executed in this block.
-    pub fork: &'a Fork,
+    pub fork: &'a mut Fork,
```

Comment:
If I remember correctly, previously we needed `&mut Fork` because Rust runtime did manage `flush` and `rollback` calls in the `before_commit` method. Do we still need it for some reason?

Reply:
Yes, it is necessary for supervisor extensions; specifically, for isolating application of config updates.